adagrams/game.py

```diff
@@ -1,11 +1,74 @@
-def draw_letters():
-    pass
+import random
+letter_set1 = ["A","I"]* 9
+letter_set2 = ["B","C","F","H","M","P","V","W","Y"] * 2
+letter_set3 = ["D","L","S","U"] * 4
+letter_set4 = ["E"] * 12
+letter_set5 = ["G"] * 3
+letter_set6 = ["J","K","Q","X","Z"]
+letter_set7 = ["N","R","T"] * 6
+letter_set8 = ["O"]* 8
+letters_list = letter_set1 + letter_set2 + letter_set3 + letter_set4\
+               + letter_set5 + letter_set6 + letter_set7 + letter_set8
+letter_score_dict ={}
+for letter in ["A", "E", "I", "O", "U","L", "N", "R","S","T"]:
+    letter_score_dict[letter] = 1
+dict_items = {"D":2,"G":2,"K":5,"J":8,"X":8,"Q":10,"Z":10}
+letter_score_dict.update(dict_items)
+for letter in ["B", "C", "M", "P"]:
+    letter_score_dict[letter]= 3
+for letter in ["F", "H", "V", "W", "Y"]:
+    letter_score_dict[letter]= 4
```

Comment:
Here we don't have to write code to build up letters_score_dict. We already have this info given to us in the README so it's fine to create a global variable that is a constant (a variable whose value will not change) like this without writing any code:

```python
LETTERS_SCORE_DICT = {
        "A" : 1,
        "B" : 3,
        "C" : 3,
        "D" : 2,
        "E" : 1,
        "F" : 4,
        "G" : 2,
        "H" : 4,
        "I" : 1,
        "J" : 8,
        "K" : 5,
        "L" : 1,
        "M" : 3,
        "N" : 1,
        "O" : 1,
        "P" : 3,
        "Q" : 10,
        "R" : 1,
        "S" : 1,
        "T" : 1,
        "U" : 1,
        "V" : 4,
        "W" : 4,
        "X" : 8,
        "Y" : 4,
        "Z": 10,
    }
```

If we have a constant, we can also avoid writing unnecessary code that could introduce a bug.